doc: Charge RPG / SRD revision (#24)
fariapp/fari-games: public/documents/fari-games/charge-srd.md

The number of dice to roll depends on the `action` the character is doing. The list of possible action can be found in the character creation section.

> In this game, NPCs don't roll for their actions. Instead, an action roll does double-duty: **it resolves the action of the PC as well as any NPCs that are involved**. The action roll tells us the results, and the consequences of an action at the same time.

Reviewer: If taking above suggestion for "NPCs always succeed unless players intervene", might make sense to carry it through to the SRD something along those lines? 

```suggestion
In this game, NPCs don't roll for their actions. NPCs succeed unless the players attempt to stop them, in which case the action roll does double-duty: **it resolves the action of the PC as well as any NPCs that are involved**. The action roll tells us the results, and the consequences of an action at the same time, encouraging players to be active participants in the story.
```

> To make an action goal, we go through the following steps:

Reviewer: ```suggestion
To make an action roll, we go through the following steps:
```

#### 4. Player Evens the Odds

> The player decides if they want to even the odds by adding bonus dice to their pool.

Reviewer: Rest of this section talks to the player with "you":

```suggestion
As the player, you decides if you want to even the odds by adding bonus dice to your pool.
```
